daydrawing.py

```diff
@@ -1,26 +1,29 @@
 # -*- coding: utf-8 -*-
 from dateutil.parser import parse
-import json
```

Comment:
Since the requests module deals with the JSON data, using it's builtin JSON decoder, there is no reason to load the json module.